conf/messages

```diff
@@ -158,6 +158,27 @@ materialsPage.warning = This does not include work related expenses (fuel, hotel
 materialsPage.error.agent = Select yes if your client paid for materials
 materialsPage.error.individual = Select yes if you paid for materials
 
+#################   Materials Amount Page   ############################
+
+materialsAmountPage.title.individual = How much did you pay for building materials for your contractor?
+materialsAmountPage.title.agent = How much did your client pay for building materials for the contractor?
+materialsAmountPage.heading.individual = How much did you pay for building materials at {0}?
+materialsAmountPage.heading.agent = How much did your client pay for building materials at {0}?
+materialsAmountPage.replayContent.p1.individual = If you did not pay {0} for materials,  tell us the correct amount.
```

Comment:
there's an extra space after the comma for `materialsAmountPage.replayContent.p1.individual`. This is for both the welsh and english message

Reply:
Fixed.